insights/parsers/sssd_conf.py

```diff
@@ -79,6 +157,10 @@ class SSSD_Config(IniConfigFile):
         >>> 'ldap_uri' in domain
         True
     """
+    def __init__(self, context):
+        deprecated(SSSD_Config, "Please use the :class:`insights.parsers.sssd_conf.SSSDConf` instead.", "3.6.0")
```

Comment:
@pbrezina - sorry for the intermittent comment to the PR, please check if we can change suggested parser in the msg with `insights.combiners.sssd_conf.SSSDConfAll`? As you also added a new combiner for "etc/sssd/sssd.conf" and "etc/sssd/conf.d/*.conf", the combiner might be the right component for user to use.

Reply:
Done.